### ISSUE TITLE
Improved YOLO Based Detection Algorithm for Floating Debris in Waterways

### ISSUE THREAD

**Issue by LornaM12** [elena-andreini/TriesteItalyChapter_PlasticDebrisDetection#47]
Abstract
Various floating debris in the waterway can be used as one kind of visual index to measure
the water quality. The traditional image processing method is difficult to meet the requirements of
real-time monitoring of floating debris in the waterway due to the complexity of the environment,
such as reflection of sunlight, obstacles of water plants, a large difference between the near and far
target scale, and so on. To address these issues, an improved YOLOv5s (FMA-YOLOv5s) algorithm
by adding a feature map attention (FMA) layer at the end of the backbone is proposed. The mosaic
data augmentation is applied to enhance the detection effect of small targets in training. A data
expansion method is introduced to expand the training dataset from 1920 to 4800, which fuses the
labeled target objects extracted from the original training dataset and the background images of
the clean river surface in the actual scene. The comparisons of accuracy and rapidity of six models
of this algorithm are completed. The experiment proves that it meets the standards of real-time
object detection.
 
Here is the[Link] to the paper(https://pmc.ncbi.nlm.nih.gov/articles/PMC8465247/pdf/entropy-23-01111.pdf)
